osf/models/identifiers.py

```diff
@@ -36,6 +40,20 @@ def delete(self):
             raise IdentifierHasReferencesError
         super().delete()
 
+    def validate_identifier_value(self):
+        # We created the PID, so assume valid
+        if self.object_id is not None:
+            return True
+
+        # If we don't know how to validate a PID, assume it's fine
+        try:
+            validator = identifier_utils.PIDValidator.for_identifier_category(self.category)
+        except NoSuchPIDValidatorError:
+            return True
```

Comment:
Minor: Should this be logged in any way, e.g. `sentry.log_message(f'Unvalidatable PID category detected: {self.category}')`

Reply:
Added sentry logging in `PIDValidator` prior to raising